src/utils/resolveRTL.js

```diff
@@ -0,0 +1,29 @@
+import { from as flatten } from 'array-flatten';
+import rtlCSSJS from 'rtl-css-js';
+
+import separateStyles from './separateStyles';
+
+// Styles is an array of properties returned by `create()`, a POJO, or an
+// array thereof. POJOs are treated as inline styles. This version of the
+// resolve function explicitly does no work to flip styles for an RTL context.
+// This function returns an object to be spread onto an element.
+export default function resolveRTL(css, styles) {
+  const flattenedStyles = flatten(styles);
+
+  const {
+    aphroditeStyles,
+    hasInlineStyles,
+    inlineStyles,
+  } = separateStyles(flattenedStyles);
+
+  const result = {};
+  if (aphroditeStyles.length > 0) {
+    result.className = css(...aphroditeStyles);
+  }
+
+  if (hasInlineStyles) {
+    result.style = rtlCSSJS(inlineStyles);
```

Comment:
Could you remind me, why isn't this already captured by the interfaceFactory's call to `rtlCSSJS` in the `createRTL` function? (sorry i'm not super familiar with how this interface works)

Reply:
Right so the `create` method deals with stylesheets, e.g. styles that you define in the thunk that gets passed to `withStyles`. Inline styles can be defined on the fly and are passed in as an object to the css method itself... they are explicitly not part of the stylesheet definition and so don't get handled by the create. That's why we have to do this here as well. :) 

(notice that the aphrodite styles are chill tho because the create took care of those)

Reply:
thanks for explaining! 👍